Movies_MVVM/Movies_MVVM/SearachFilmScreen/ViewModel/SearchFilmViewModel.swift

```diff
@@ -0,0 +1,61 @@
+// SearchFilmViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Search film view model
+final class SearchFilmViewModel: SearchFilmViewModelProtocol {
+    // MARK: - Public Properties
+
+    var networkService: NetworkServiceProtocol
+    var imageService: ImageServiceProtocol
+    var movies: [Movie]?
+    var searchFilmStates: ((SearchFilmState) -> ())?
```

Comment:
```suggestion
    var searchFilmStates: 
```
тоже тайпэлиас сделай